docs/main/updating/5-0.md

```diff
@@ -0,0 +1,152 @@
+---
+title: Updating to 5.0
+description: Guide for updating Capacitor from v4 to v5 in your app
+slug: /updating/5-0
+---
+
+# Updating from Capacitor 4 to Capacitor 5
+
+Compared to previous upgrades, the breaking changes between Capacitor 4 and 5 are extremely minimal. In this guide, you'll find steps to update your project to the current Capacitor 5 version as well as a list of breaking changes for our official plugins.
+
+## Using the CLI to Migrate
+
+Install the latest version of the Capacitor CLI to your project using `npm i -D @capacitor/cli@latest`. Once installed, simply run `npx cap migrate` to have the CLI handle the migration for you. If any of the steps for the migration are not able to be completed, additional information will be made available in the output in the terminal. The steps for doing the migration manually are listed out below.
+
+## iOS
+
+The following guide describes how to upgrade your Capacitor 4 iOS project to Capacitor 5.
+
+### Upgrade Xcode
+
+Capacitor 5 requires Xcode 14.1+.
+
+### Update .gitignore
+
+Make the following changes to your `.gitignore` file:
+
+```diff
++ App/output 
+- App/Podfile.lock
+```
+
+### Update Assets to use a single app icon
+
+Xcode 14 supports a single app icon of 1024x1024, so you can clean up your AppIcon.appiconset by removing all unnecessary sizes.
```

Comment:
maybe explain how to do it?
If you choose Single Size from this prompt, it automatically deletes all the unneeded icons.
<img width="882" alt="Captura de pantalla 2023-03-31 a las 11 20 59" src="https://user-images.githubusercontent.com/1637892/229080658-92b819ea-7b06-42dc-9478-820cc5efb9f0.png">